<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNote</class>
 <widget class="QDialog" name="DialogNote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>382</width>
     <height>282</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Historique :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="displayHistorique"/>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Note (/10):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="noteInput"/>
    </item>
    <item>
     <widget class="QPushButton" name="buttonMettreNote">
      <property name="text">
       <string>Mettre la note</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
